src/components/ProbeEditor.tsx

```diff
@@ -110,6 +114,14 @@ const ProbeEditor = ({ probe, onReturn }: Props) => {
           <div>
             <Legend>{legend}</Legend>
             <Container margin="md">
+              {probe.public && (
```

Comment:
nit: in old UI turned off public meant users know that probe is private.

should we show text for private probes as well.

I am thinking something like:

```
This probe is Private
Private probes are operated by your organization and can only run your checks.
```